azext_iot/_params.py

```diff
@@ -819,6 +819,18 @@ def load_arguments(self, _):
             type=int,
             help="Maximum number of configurations to return. By default all configurations are returned.",
         )
+        context.argument(
+            "custom_metric_queries",
+            nargs="+",
+            options_list=["--custom-metric-queries"],
```

Comment:
Any short aliases? For here and custom labels below.

Reply:
can I use the first letters for short alias? like --csq and --cl?😀

Reply:
Yah